packages/codemirror/src/editor.ts

```diff
@@ -615,7 +631,17 @@ class CodeMirrorEditor implements CodeEditor.IEditor {
       return;
     }
     this._changeGuard = true;
-    this._model.value.text = this.doc.getValue();
+    let value = this._model.value;
+    let start = doc.indexFromPos(change.from);
```

Comment:
I think this is the line that is brittle, because it `change.from` is "in the pre-change coordinate system" according to the manual: https://codemirror.net/doc/manual.html